### ISSUE TITLE
Feature Request: Count physical money by notes/coins 

### ISSUE THREAD

**Issue by alex9099** [mtotschnig/MyExpenses#669]
It would be nice to be able to register how many (and what value) of coins/bills i have in my wallet, transactions could also be able to have their "value" determined by coins/notes count

**Comment by mtotschnig** [comment]
This sounds interesting, but I am afraid that the effort of development would be higher than the number of users needing this feature.
Could you explain the use case and value in more detail?

**Comment by alex9099** [comment]
I guess this would be just a nice feature to have, i don't have a specific use case, perhaps it would be useful to remember better some transaction that i forgot to register (as coins/notes can make specific combinations, for example to buy something 1.5€ i can use 1€+.20+.20+.10, so when i do reconciling if those are missing i can be somewhat sure that i used them on that specific transaction, though this wouldn't work if there is a huge amount of coins/notes missing)

**Comment by L4rsw** [comment]
> It would be nice to be able to register how many (and what value) of coins/bills i have in my wallet, transactions could also be able to have their "value" determined by coins/notes count

I created an extra account named cash and it shows me how much money i have in my wallet.  Just as a hint :)

**Comment by alex9099** [comment]
Yeah, same as i do. Perhaps i lose money often or simply don't register the transactions. Some times when i check to make sure everything is ok i even have more money on the wallet than i have registered :+1: 

Either way now that i think about this issue, it was just a shower thought, not that useful, so yeah, priority to do it would be near 0 (IMO)

**Comment by L4rsw** [comment]
@alex9099 Your are very lucky, when I check my wallet and the app, it#s always less money in the wallet than the app shows me

**Comment by alex9099** [comment]
Adding to this, if this feature gets implemented, a even more niche feature somewhat related would be to record the bills serial number.

The scenario that lead me to think about this was because I deposited some bills on an ATM and it ate the bills and didn't count them, to make tracking the bills easier (proving that the notes on the ATM were actually in my possession)

Also it would help to track bills for the eurobilltracker (not sure if they have any app or something like that)
